### PR TITLE
fix(forms): Phone number merged with country code, bugfixes

### PR DESCRIPTION
**Pull request by henit**

### CODE REVIEW

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/feature-fields/PhoneNumber/properties.mdx [hunk 1]
```diff
@@ -15,6 +15,7 @@ import DataValueReadwriteProperties from '../../data-value-readwrite-properties.
 | `help`                                  | `object`            | _(optional)_ Provide a help button. Object consisting of `title` and `contents`                             |
 | `countryCodeFieldClassName`             | `string`            | _(optional)_ Class name for the country code component.                                                     |
 | `numberFieldClassName`                  | `string`            | _(optional)_ Class name for the number component.                                                           |
+| `counttryCodePlaceholder`               | `string`            | _(optional)_ Placeholder for the country code field.                                                        |
```
Comment (tujoworker) [inline packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/feature-fields/PhoneNumber/properties.mdx:18]:
```suggestion
| `countryCodePlaceholder`               | `string`            | _(optional)_ Placeholder for the country code field.                                                        |
```

File: packages/dnb-eufemia/src/extensions/forms/Field/PhoneNumber/PhoneNumber.tsx [hunk 2]
```diff
@@ -13,6 +14,7 @@ export type Props = FieldHelpProps &
   FieldProps<string, undefined> & {
     countryCodeFieldClassName?: string
     numberFieldClassName?: string
+    counttryCodePlaceholder?: string
```
Comment (tujoworker) [inline packages/dnb-eufemia/src/extensions/forms/Field/PhoneNumber/PhoneNumber.tsx:17]:
```suggestion
    countryCodePlaceholder?: string
```

File: packages/dnb-eufemia/src/extensions/forms/Field/PhoneNumber/style/dnb-phone-number.scss [hunk 3]
```diff
@@ -1,20 +1,8 @@
...
-    width: 100%;
-  }
-
-  &--width-large {
-    width: var(--forms-field-width--large);
-  }
-
   &__country-code {
-    flex: 0 9rem;
+    width: 9rem;
```
Comment (tujoworker) [inline packages/dnb-eufemia/src/extensions/forms/Field/PhoneNumber/style/dnb-phone-number.scss:5]: Now, we have used `9rem` before in this PR, but is does it have a relation to one of these field sizes? `--forms-field-width--medium` – could 

```scss
calc(var(--forms-field-width--medium) - 2rem) // with a comment why -2rem
```

do the same?

File: packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/feature-fields/PhoneNumber/properties.mdx (hunk 1 above)
Comment (langz) [inline packages/dnb-design-system-portal/src/docs/uilib/extensions/forms/feature-fields/PhoneNumber/properties.mdx:18]:
```suggestion
| `countryCodePlaceholder`               | `string`            | _(optional)_ Placeholder for the country code field.                                                        |
```

**Review by tujoworker — COMMENTED** [review]
Nice work! I just have some comments to outline as first step.

**Comment by tujoworker** [comment]
Also, there is a conflict with `CountryCode.tsx` in this PR.

**Comment by langz** [comment]
I added a commit, https://github.com/dnbexperience/eufemia/pull/2755/commits/25d8ca505d2b7aab84d9700a5512b24cb859b4d7,  renaming `counttryCodePlaceholder` to `countryCodePlaceholder`.

**Comment by henit** [comment]
> I added a commit, [25d8ca5](https://github.com/dnbexperience/eufemia/commit/25d8ca505d2b7aab84d9700a5512b24cb859b4d7), renaming `counttryCodePlaceholder` to `countryCodePlaceholder`.

Ah, nice. I added that one to check if you where paying attention 🤣

**Review by tujoworker — APPROVED** [review]

**Pull request merged**